src/main/resources/avro/bdg.avdl

```diff
@@ -363,12 +363,55 @@ record NucleotideContigFragment {
  duplicated sequence.
  */
 enum StructuralVariantType {
+
+  /**
+   Breakend.  VCF INFO reserved key "SVTYPE" value "BND".
+   */
+  BREAKEND,
+
+  /**
+   Copy number variable region (may be both deletion and duplication).
+   VCF INFO reserved key "SVTYPE" value "CNV".
+   */
+  COPY_NUMBER_VARIABLE,
```

Comment:
Actually, it's not too bad: the crux is "this region has varied copy number across samples". A "copy number variant" would be the specific change in copy number. I actually think this is the right call.